service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -336,6 +336,9 @@ public String cloneWorkspace(
     String jobDescription =
         String.format("Clone workspace: name: '%s' id: '%s'  ", workspaceName, workspaceUuid);
 
+    destinationWorkspace =
```

Comment:
Instead of doing this here, we should do this in the controller where we are constructing the original destination workspace object.

In the current code base, `WorkspaceApiController.cloneWorkspace()` does not have the source workspace object. However, after Zach's code change goes in, it will. So I would suggest either waiting until Zach's change goes in and then reworking this so it is in the controller. Alternately, you could leave this as is with a TODO and make yet another ticket to fix it after Zach's change goes in.